lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2479,6 +2499,25 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(
   {
     LLDB_SCOPED_TIMERF("%s (getStdlibModule)", m_description.c_str());
     const bool can_create = true;
+
+    // Report progress on module importing by using a callback function in
+    // swift::ASTContext
+    Progress progress("Importing Swift standard library modules");
```

Comment:
Same here, drop the "modules" part.

```suggestion
    Progress progress("Importing Swift standard library");
```

Reply:
Not sure how I missed both of these, thanks for the correction!